hw/ip/spi_device/doc/_index.md

```diff
@@ -677,15 +677,25 @@ If `invalid_locality` configuration is set, the logic returns `INVALID` value to
 If the request is in the supported locality (0-4), the logic checks `TPM_ACCESS_x.activeLocality` then returns data based on the table 39 in the spec for Return-by-HW registers.
 Other registers in the table should be processed by SW.
 
+## Countermeasures
+
+This version of the SPI_DEVICE IP implements the parity to detect bit flip errors on the internal SRAM.
+The HW checks the parity error when the SW reads data from the SRAM.
+The error is reported to the SW via TL D channel error signal.
```

Comment:
perhaps also mention this error doesn't trigger any alert.

Reply:
I don't think it is needed as there's alert list table in the spec.

Reply:
all right, sounds good.

Reply:
hmm... i dont think strictly speaking this is a countermeasure. Because if it is, we technically need to document this as `sec_cm` and go through a v2s process... this is more for reliability in general right?

Reply:
yeah. You pointed the thing I thought about the section title. I could not decide if the title should include "Errors" other than "Countermeasures"

Reply:
Could not think of better section title. renamed to `Detecting Reliability Errors`